table.go

```diff
@@ -0,0 +1,75 @@
+package heptane
```

Comment:
A question raises here... should we document packages as we do at func level?

Reply:
Oh, yes, we should.